extensions/wikia/UserRenameTool/maintenance/RenameUser_local.php

```diff
@@ -0,0 +1,130 @@
+<?php
+/**
+ * @package MediaWiki
+ * @subpackage Maintanance
+ *
+ * @author: Władysław Bodzek <wladek@wikia-inc.com>
+ *
+ * @usage: SERVER_ID=177 php RenameUser_local.php --conf /usr/wikia/conf/current/wiki.factory/LocalSettings.php --aconf /usr/wikia/conf/current/AdminSettings.php
+ */
+
+ini_set( "include_path", dirname( __FILE__ ) . "/../../../../maintenance" );
+
+$options = [ 'help' ];
+
+// Used by commandLine.inc
+$optionsWithArgs = [
+	'rename-user-id',
+	'rename-old-name',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "rename-old-name" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)